### DISCUSSION TITLE
v3.2.0

### DISCUSSION THREAD

**Discussion by osy** [osy86/HaC-Mini#1015] [Releases]
## Changes
* Support macOS Ventura
* Add FeatureUnlock kext to support AirPlay mirroring on >= 12.3 (#991) (thanks @goodbest)
* Add AppleIGB kext for LAN2 on >= 12.0 (#947)
* Add timeout for Brcm wifi cards to fix boot issues on >= 12.1 (thanks @goodbest)
* Removed manual UI scale option (OpenCore now detects this automatically)
* Enabled OpenCore support for booting Linux directly

## Driver Updates

* AirportBrcmFixup 2.1.4 -> 2.1.6
* AppleALC 1.7.1 -> 1.7.3
* BrcmPatchRAM 2.6.1 -> 2.6.3
* FeatureUnlock 1.0.8 -> 1.0.9
* Lilu 1.6.0 -> 1.6.1
* NVMeFix 1.0.9 -> 1.1.0
* OpenCore 0.8.0 -> 0.8.2
* Polaris22Fixup v1.3.5 -> v1.3.6
* VirtualSMC 1.2.9 -> 1.3.0
* WhateverGreen 1.5.8 -> 1.6.0

## Known Issues
* OpenIntelWireless does not have a stable build for Ventura and therefore is not included in the package

## Messages
* **Important**: Apple has started requiring Secure Boot for updating to macOS Monterey 12.2 and above (including to macOS Ventura). See the [previous release](https://github.com/osy/HaC-Mini/releases/tag/v3.1.5) for details.
* [As announced a few months ago](https://github.com/osy/HaC-Mini/discussions/578), this project will be transitioning to a long-term support model. macOS Ventura will likely be the last supported version.

<hr /><em>This discussion was created from the release <a href='https://github.com/osy/HaC-Mini/releases/tag/v3.2.0'>v3.2.0</a>.</em>

**Comment by kingrhy** [comment]
upgrade to macos13 newest beta version smoothly and working good.

**Reply by hoangtuan21193** [reply]
@kingrhy hi,
Are you using Nuc8i7hvk?

**Reply by kingrhy** [reply]
YES, HOW CAN I HELP YOU?

**Reply by kingrhy** [reply]
For macos13 beta version i mentioned before. please notice: after use some days. its crash suddenly without any warning , one day morning when i reach to office and press keyboard try to wake up it. then no any response. i try to re-booting with u-key and re-install pkg . and power off and repower it. its does not work. then i reinstall macos12.5 directly. its back.    so far still wait official 3.20 release and macos 13 official release.

**Reply by hoangtuan21193** [reply]
@kingrhy 
Thank you,
I'm using 3.20 and install macos12.6 but macOS shutdown automatically after 1 min or sometime when I leave it over night
Does this happen to you?

My issue is here:
https://github.com/osy/HaC-Mini/issues/1021

**Reply by kingrhy** [reply]
no its not happened to me. did you try to use hackintool to fix power(you can see the flash icon in a cycle when you open this tool).  see attach file. i am not sure if that could help you .but you can try if you  are did not do that yet.
<img width="656" alt="hakcintool" src="https://user-images.githubusercontent.com/16399048/195473159-94f02d80-3f6f-4458-84fa-e8fd6dddf978.png">

**Reply by kingrhy** [reply]
BTW,  i use the 3.15 version pkg. if above method cannot fix. you can try save efi and use 3.15 pkg make a u-disk try to boot up and test for one or two days to isolate possible version issue..  if use 3.15 ok. then you can copy efi to boot disk.

**Reply by hoangtuan21193** [reply]
@kingrhy 
Thank you so much.
I will try.

**Comment by Ericanymouse** [comment]
PciRoot(0x0)/Pci(0x2,0x0) AAPL,ig-platform-id：0300C89B, then the Sidecar will work.

**Reply by osy** [reply]
Is sidecar not working otherwise? Was it broken recently?

**Reply by osy** [reply]
We use https://github.com/acidanthera/FeatureUnlock

**Reply by Ericanymouse** [reply]
Yes, the sidecar is not working with the default settings. And the Mac OS is Monterey 12.5

**Reply by Ericanymouse** [reply]
emmmmm, the Universal Control is also not working with the default settings.

**Reply by Ericanymouse** [reply]
I reinstall the pkg and choose for Secure Boot, and now I found sidecar and Universal Control are working now....
That's weird！
Maybe it is related to replacement of the Apple Wifi card after the first install.

**Comment by kevinpvo** [comment]
I currently have big sur mac os 11 installed without secure boot enabled - and i wish to upgrade to mac os 12.5 or 13 in near future.

Could I ask you guys for some clarification / confirmation on the step by step process for enabling secure boot during upgrade?

- Do I perform the following steps in this particular order:
1. First reinstall the hac-mini pkg with secure boot enabled while on mac os 11.
2. Then perform OTA upgrade from mac os 11 to 12.
3. Then go into recovery and "bless --folder "/Volumes/Macintosh HD/System/Library/CoreServices" --bootefi --personalize"
4. Boot into mac os 12

- Also will I need to enable secure boot in the bios (via intel visual bios) at any point in the upgrade process?

Thank you so much - i've been slightly confused on the exact steps to enable secure boot and perform the OTA upgrade, so i figured I would ask the community for some clarification and help. And also I want to say thank you @osy for an amazing job with the project and support you have provided to all of us!

**Comment by Medizinmann-MD** [comment]
Yes to 1. then you must do 3. then you can got for 2.

Or in other word: Yes, you reinstall latest hac-mini pkg with secure boot enabled and then you will need to reboot into recovery and personalize your system - afterwards you can to the OTA Update.

And no - you don't want do enable secrue boot in intel bios.

**Reply by kevinpvo** [reply]
Thank you so much for the clarification!!
